src/common/cockpittransport.c

```diff
@@ -165,6 +165,27 @@ cockpit_transport_emit_recv (CockpitTransport *transport,
 }
 
 void
+cockpit_transport_emit_control (CockpitTransport *transport,
```

Comment:
Should cockpit_transport_default_recv() call this function? Otherwise we have code duplication.